docs/modules/reference/pages/response-bodies.adoc

```diff
@@ -23,6 +24,14 @@ type of the value in the :body key of the xref:response-map.adoc[].
 | {core_async} channel | nil | see below | yes
 |===
 
+[#nio-channel]
+== nio Channel
+When the response body is a ByteBuffer or an NIO Channel, then Pedestal will convert
```

Comment:
Why include ByteBuffer as well? We should probably only do it for Channel (there is no async operation to read from a ByteBuffer, as it is already mapped to memory)

Reply:
I'm not sure, just maintaining parity with a lot of stuff that predates me.  
https://github.com/pedestal/pedestal/blob/a7ce6c0f5da45b7db45c9dc7aded9e80da808b4e/service/src/io/pedestal/http/impl/servlet_interceptor.clj#L148

https://github.com/pedestal/pedestal/blob/a7ce6c0f5da45b7db45c9dc7aded9e80da808b4e/service/src/io/pedestal/http/container.clj#L30

https://github.com/pedestal/pedestal/blob/a7ce6c0f5da45b7db45c9dc7aded9e80da808b4e/jetty/src/io/pedestal/http/jetty/container.clj#L48

Reply:
So that looks like Jetty, at least, will support ByteBuffer to async operations.